### PR TITLE
Аксенов - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by Praice37**

### CODE REVIEW

**Comment by Praice37** [comment]
@ArtemSadikov @alinaut

**Comment by ArtemSadikov** [comment]
:+1:

**Comment by alinaut** [comment]
:+1:

**Comment by Praice37** [comment]
@alexbatashev

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Aksenov Nikita
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Aksenov_Nikita_ComplexNumberTest,
+    Can_Not_Divide_By_Zero) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/master/modules/complex-number/test/test_complex_number.cpp#L114

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    // Act
+    ComplexNumber z1(re1, im1);
+    ComplexNumber z2(re2, im2);
+
+    // Assert
+    EXPECT_ANY_THROW(z1 / z2);
+}
+
+TEST(Aksenov_Nikita_ComplexNumberTest,
+    Two_Reverse_Numbers_Are_Not_Equal) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/master/modules/complex-number/test/test_complex_number.cpp#L140

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    // Act
+    ComplexNumber z1(re1, im1);
+    ComplexNumber z2(re2, im2);
+
+    // Assert
+    EXPECT_NE(z1, z2);
+}
+
+TEST(Aksenov_Nikita_ComplexNumberTest,
+    The_Same_Result_After_Multiplication_By_Certain_Number) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: неудачное имя для теста

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    // Act
+    ComplexNumber sum1 = (z1 + z2) + z3;
+    ComplexNumber sum2 = (z2 + z3) + z1;
+
+    // Assert
+    EXPECT_EQ(sum1, sum2);
+}
+
+TEST(Aksenov_Nikita_ComplexNumberTest,
+    Can_Subtract_Two_Negative_Numbers) {
```
Comment (alexbatashev) [inline modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/master/modules/complex-number/test/test_complex_number.cpp#L75

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp (hunk 3 above)
Comment (Praice37) [reply modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: Changed the name of the test

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp (hunk 1 above)
Comment (Praice37) [reply modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: Wrote another test

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp (hunk 2 above)
Comment (Praice37) [reply modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: Wrote another test

File: modules/complex-number/test/test_aksenov_nikita_complex_number.cpp (hunk 4 above)
Comment (Praice37) [reply modules/complex-number/test/test_aksenov_nikita_complex_number.cpp]: Wrote another test

**Comment by Praice37** [comment]
@alexbatashev

**Pull request merged**